### DISCUSSION TITLE
getShopReceipts - include cancelled/refunded receipts in the response

### DISCUSSION THREAD

**Discussion by nmrdxt** [etsy/open-api#814] [Ideas]
Currently, getShopReceipts does not include cancelled orders in the returned response – it would be great if there was a way to include cancelled receipts as well (or possible include a flag that would allow including cancelled receipts).

Thanks!

**Comment by Gareth064** [comment]
Are you sure they aren't being returned?

The way to identify a cancelled order is by looking for refund information which is a nested object on each Receipt
![image](https://user-images.githubusercontent.com/22307223/211854016-a3a6c2e8-8ed5-45d2-9f90-d503ba0388b0.png)

If they aren't being returned then feels like this refunds object is partially pointless.

**Reply by nmrdxt** [reply]
The receipts are not included in the response. I've tested multiple scenarios, they are simply excluded from the returned receipts.

**Comment by Gareth064** [comment]
You could also chip in here where I have been requesting for better ways to detect "post sales" activities, like refunds and cancellations.
#701
